tripgo.swagger.yaml

```diff
@@ -89,6 +89,223 @@ paths:
+            properties:
+              region:
+                type: string
+                description: Region name/code from `regions.json`.
+              modes:
+                type: array
+                items:
+                  type: string
+                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
+              operator:
+                type: string
+                description: Operator name to retrieve information from. When ´full´ parameter is false, this field is ignored. THIS COULD BE AN ARRAY TOO.
+              onlyRealTime:
+                type: boolean
+                description: Boolean to filter only operators with Real Time support. Only valid when ´full´ parameter is false. When true, returned operator list will contain only TSP with Real Time service.
+              full:
+                type: boolean
+                description: Boolean if result should include full information for operator/s or just a list with all their names ignoring ´mode´ array in response. False value by default.
+            required:
+              - region
+            example:
+              - region: US_CA_LosAngeles
+                modes:
+                  - pt_pub_tram
+                  - pt_pub_bus
+      responses:
+        200:
+          description: Successful response
+          schema:
+            type: array
+            items:
+              type: object
+              description: TSP information.
+              properties:
+                name:
+                  type: string
+                  description: TSP name.
+                mode:
+                  type: array
+                  items:
+                    description: Detailed information for this operator under a specific public transport mode.
+                    type: object
+                    properties:
+                      numberOfServices:
+                        type: integer
+                        description: Total number of services for this operator and mode.
+                      realTime:
+                        type: object
+                        $ref: '#/definitions/RealtimeSnapshot'
+            example:
+              code: ONCE WE HAVE RESPONSE MODEL DEFINED
+        400:
+          description: SHOULD I PUT ERROR MODES FOR SPECIFIC SCENARIOS LIKE NON-VALID FILTER USED, ETC?
```

Comment:
I don't think we need that, but the error response should also be a JSON that includes a description of what's wrong. See the TTP servlet spec for an example.